CoreScriptsRoot/ServerCoreScripts/ClientChat/ChannelsBar.lua

```diff
@@ -281,6 +283,12 @@ function methods:CreateTweeners()
 	
 end
 
+function methods:WaitUntilParentedCorrectly()
```

Comment:
Is there no way to make this function once? I'm worried someday either we will need to tweak it or change something about it and someone will forget to update one of these.